packages/wrangler/src/api/remoteBindings/start-remote-proxy-session.ts

```diff
@@ -43,7 +44,11 @@ export async function startRemoteProxySession(
 				port: await getPort(),
 			},
 			inspector: false,
-			logLevel: "error",
+			logLevel:
+				// If the user has specified WRANGLER_LOG to "debug", the are likely trying to debug some issue,
+				// so we should respect that here as well. In any other case, to avoid noisy logs, we just simply
+				// fall back to "error"
+				getLogLevelFromEnv() === "debug" ? "debug" : "error",
```

Comment:
This is just an idea.... another idea could be to introduce a new env variable like `WRANGLER_REMOTE_SESSION_LOG` to control this `logLevel` 😕 

(PS: Here setting `logLevel` to the `WRANGLER_LOG` set by the user, can majorly backfire in my opinion since users could end up getting lots of unexpected and noisy logs in this way 😓)